### PR TITLE
Init impl of messages tab

### PR DESCRIPTION
**Pull request by ysabri**

### CODE REVIEW

File: src/components/Messages/Messenger/Messenger.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  background: white;
+  grid-row-start: 1;
+  grid-row-end: span 3;
+}
+
+.message-content {
+  background: white;
+  grid-row-start: 1;
+  grid-row-end: span 3;
+  height: 81.9vh;
```
Comment (ysabri) [inline src/components/Messages/Messenger/Messenger.css:40]: 🔎

File: src/components/Header/HeaderBar.js [hunk 2]
```diff
@@ -55,6 +56,7 @@ class HeaderBar extends React.Component {
           <ProtectedRoute exact path='/' loggedIn={ this.state.loggedIn } currUser={ this.state.currUser } component={HomePage} />
           <Route path="/dashboard" render={(props) => <Dashboard {...props} currUser={ this.state.currUser} />} />
           <Route path="/settings" component={AccountOptionsComponent}/>
+          <Route path="/Messages" render={(props) => <Messenger {...props} currUser={this.state.currUser}/>}/>
```
Comment (kalshafei) [inline src/components/Header/HeaderBar.js:59]: Change to lower case 'messages'

File: src/components/Messages/ConversationList/index.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ const getConversations = () => {
+      // getting convos from server goes here
+        let newConversations = [
+          {
+            photo: "https://avatars0.githubusercontent.com/u/19934651?s=460&u=63e4e3b3f4e3e5b8a0ce31eea96b2d0a9e4d5d46&v=4",
+            name: "Janky Leeroy",
+            text: "I hate React"
+          },
+        {
+            photo: "https://giantbomb1.cbsistatic.com/uploads/scale_small/0/1976/224310-cv_sotn_succubus.jpg", // don't visit this site
```
Comment (kalshafei) [inline src/components/Messages/ConversationList/index.js:24]: as if i wouldnt

**Review by kalshafei — CHANGES_REQUESTED** [review]

File: src/components/Messages/ConversationList/index.js (hunk 3 above)
Comment (ysabri) [reply src/components/Messages/ConversationList/index.js:24]: Pls report back on your findings

**Pull request merged**